app/assets/javascripts/controllers/navigation/active_item_controller.js

```diff
@@ -0,0 +1,36 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class ActiveItemController extends Controller {
+  static targets = ["menu"];
+  static values = {
+    cssClass: { type: String, default: "active" },
+    addToParents: Boolean
+  }
+
+  connect() {
+    console.error("connect")
+    const item = this.getParent(this.activeLink)
+    this.setActiveItem(item);
```

Comment:
What will happen if this controller is connected multiple times because a turbo navigation has occurred?

Consider adding `disconnect` to remove the active classes.